runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -248,7 +252,31 @@ ROMClassBuilder::buildROMClass(ROMClassCreationContext *context)
 	context->recordLoadEnd(result);
 	return result;
 }
+BuildResult
+ROMClassBuilder::injectInterfaces(ClassFileOracle *classFileOracle)
+{
+	PORT_ACCESS_FROM_PORT(_portLibrary);
+	if (NULL == _interfaceInjectionInfo) {
+		_interfaceInjectionInfo = (InterfaceInjectionInfo *) j9mem_allocate_memory(sizeof(InterfaceInjectionInfo), J9MEM_CATEGORY_CLASSES);
```

Comment:
this allocation can be avoided entirely since the number of interfaces is known statically